ecosystem/Editor-and-tool-support.md

```diff
@@ -37,12 +37,17 @@ Spacemacs users can just use the [`purescript` layer](https://github.com/syl20bn
 
 ## Build tools and package managers
 
-- [Pulp](https://github.com/purescript-contrib/pulp) - a standalone build system for PureScript ([pulp](https://www.npmjs.com/package/pulp) in `npm`)
+Maintained and actively supported tools:
+- [spago](https://github.com/purescript/spago) - PureScript package manager and build tool powered by [Dhall](https://github.com/dhall-lang/dhall-lang) and [package-sets](https://github.com/purescript/package-sets)
 - [psc-package](https://github.com/purescript/psc-package) - A package manager for PureScript based on the concept of package sets
-- [spago](https://github.com/spacchetti/spago) - PureScript package manager and build tool powered by [Dhall](https://github.com/dhall-lang/dhall-lang) and [package-sets](https://github.com/purescript/package-sets)
-- [Gulp task](https://github.com/purescript-contrib/gulp-purescript) (`gulp-purescript` in `npm`)
-- [psvm-js](https://github.com/ThomasCrvsr/psvm-js) - PureScript Version Manager
 - [purs-loader](https://github.com/ethul/purs-loader/) - PureScript loader for WebPack
+- [pscid](https://github.com/kRITZCREEK/pscid) is a lightweight file-watcher/testrunner for PS projects providing instant-rebuilds in an editor agnostic way
+
+Deprecated:
+- [Pulp](https://github.com/purescript-contrib/pulp) - a standalone build system for PureScript ([pulp](https://www.npmjs.com/package/pulp) in `npm`). *Spago is recommended over Pulp.*
+- [Gulp task](https://github.com/purescript-contrib/gulp-purescript) (`gulp-purescript` in `npm`). *Spago is recommended over the PureScript Gulp task.*
+
+Unmaintained:
+- [psvm-js](https://github.com/ThomasCrvsr/psvm-js) - PureScript Version Manager
 - [psc-pane](https://github.com/anttih/psc-pane) - Auto reloading compiler which formats a single error to fit the window
 - [purescript-psa](https://github.com/natefaubion/purescript-psa) - A pretty, flexible error/warning reporting frontend for `psc` featuring colours, original source spans in errors, warning filtering and persistence.
```

Comment:
`psa` is not unmaintained, it just does everything it needs to and does it well. It's the first thing we recommend when users ask for better error output and configuration of warnings.

Reply:
Additionally, `psa` is backed by `psa-utils` which `pscid` and the lsp implementations rely on.